Makefile

```diff
@@ -17,11 +17,32 @@
 .PHONY: setup
 setup:
 	python3 -m pip install --upgrade pip
-	python3 -m pip install a6pluginprotos --ignore-installed
-	python3 -m pip install minicache --ignore-installed
-	python3 -m pip install click --ignore-installed
-	python3 -m pip install pytest --ignore-installed
+	python3 -m pip install -r requirements.txt --ignore-installed
+
 
 .PHONY: test
 test:
-	python3 -m pytest -v tests
+	pytest --version || python3 -m pip install pytest-cov
+	python3 -m pytest --cov=apisix/runner tests
```

Comment:
I just look at the coverage.
84% is a great score to share. Shall we upload it to codecov.io and put a badger on the README? We can do it in the next PR.

Reply:
OK, good idea